adapters/repos/db/replication.go

```diff
@@ -319,3 +320,32 @@ func (s *Shard) reinit(ctx context.Context) error {
 
 	return nil
 }
+
+func (db *DB) OverwriteObjects(ctx context.Context,
+	objects []*objects.VObject,
+) ([]int64, error) {
+	result := make([]int64, len(objects))
+	for i, vobj := range objects {
+		found, err := db.Object(ctx,
+			vobj.Object.Class().String(), vobj.Object.ID(),
+			nil, additional.Properties{}, nil)
+		if err != nil {
+			return nil, err
+		}
+		// the db's stored object is not the most recent version.
+		// in this case, we overwrite it with the more recent one
+		if found.Object().LastUpdateTimeUnix < vobj.Version {
```

Comment:
I think this should be `found.Object().LastUpdateTimeUnix == vobj.Version` because obj.Version contains the previous version from coordinator perspective. If the object has not been changed in the meantime then `LastUpdateTimeUnix==Version`

Reply:
done